iceoryx-sys/build.rs

```diff
@@ -28,16 +43,17 @@ fn make_and_install(source_dir: &str, build_dir: &str, install_dir: &str) -> std
             ));
         }
 
+        let mut cmake_args = extra_cmake_args.clone();
+        cmake_args.push(format!("-DCMAKE_BUILD_TYPE=Release"));
+        cmake_args.push(format!("-DBUILD_SHARED_LIBS=OFF"));
+        cmake_args.push(format!("-DROUDI_ENVIRONMENT=ON"));
```

Comment:
This leads to clippy warnings. Please replace `format!("foo")` with `"foo.into()"`